tests/e2e/simple.spec.ts

```diff
@@ -128,3 +128,64 @@ test('does not monetize when continuous payments are disabled', async ({
     });
   });
 });
+
+test('does not monetize when global payments toggle in unchecked', async ({
+  page,
+  popup,
+  i18n,
+}) => {
+  const walletAddressUrl = process.env.TEST_WALLET_ADDRESS_URL;
+  const monetizationCallback = await setupPlayground(page, walletAddressUrl);
+
+  await test.step('disables extension', async () => {
+    await popup
+      .getByRole('checkbox', { name: 'Disable extension' })
+      //TO DO: remove force; normally this should not be necessary
+      .uncheck({ force: true });
+
+    await expect(popup.getByTestId('not-monetized-message')).toHaveText(
+      i18n.getMessage('app_text_disabled'),
+    );
+  });
+
+  await test.step('check extension payments do not go through', async () => {
+    await expect(
+      popup.getByRole('button', { name: 'Send now' }),
+    ).not.toBeVisible();
+    await expect(monetizationCallback).toHaveBeenCalledTimes(0);
+  });
+
+  await test.step('and does not monetize even with continuous payments toggle on/off', async () => {
+    const settingsLink = popup.locator(`[href="/settings"]`).first();
+    await settingsLink.click();
+
+    await popup.bringToFront();
+    await popup.getByRole('tab', { name: 'Rate' }).click();
+    const continuousPaymentsToggle = popup.getByTestId(
+      'continuous-payments-toggle',
+    );
+    await continuousPaymentsToggle.uncheck({ force: true });
+    await expect(monetizationCallback).toHaveBeenCalledTimes(0);
+
+    await expect(
+      popup.getByRole('tabpanel', { name: 'Rate' }).locator('p'),
+    ).toContainText('Ongoing payments are now disabled');
+
+    await continuousPaymentsToggle.check({ force: true });
+    await expect(monetizationCallback).toHaveBeenCalledTimes(0);
+  });
+
+  await test.step('checking global payments toggle re-enables payments in extension', async () => {
+    const backHomeLink = popup.locator(`[href="/"]`).first();
+    await backHomeLink.click();
+
+    await popup.bringToFront();
+    await popup
+      .getByRole('checkbox', { name: 'Enable extension' })
+      //TO DO: remove force; normally this should not be necessary
+      .check({ force: true });
+
+    await expect(monetizationCallback).toHaveBeenCalledTimes(1);
```

Comment:
Let's add a 2s wait here
```suggestion
    await page.waitForTimeout(2000);
    await expect(monetizationCallback).toHaveBeenCalledTimes(1);
```